ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -310,6 +313,10 @@ function test_execution() {
             export E2E_RUN_TAGS="${E2E_RUN_TAGS} and @console"
             test_execution_ui
             ;;
+        ui_destructive)
+            export E2E_RUN_TAGS="${E2E_RUN_TAGS} and @console and @destructive"
```

Comment:
This line is not needed, as we have done the same in function `test_execution_ui_destructive`

Reply:
good catch, will update